R/helpers_ggcoefstats.R

```diff
@@ -841,18 +848,33 @@ bf_meta_message <- function(data,
       dplyr::mutate(.data = ., term = as.character(term))
   }
 
+  # check defintion of priors for d and tau
+  # Note: "d.par" and "tau.par" are deprecated in metaBMA (>= 0.6.1)
+  if (class(d) == "character"){
+    d <- metaBMA::prior(family = d, param = d.par)
+  } else if (!class(d) == "prior"){
+    stop("The argument 'd' must be ",
+         "\n  (A) a character such as 'norm' specifying the family of prior distribution",
+         "\n  (B) a prior distribution specified via metaBMA::prior()")
+  }
+  if (class(tau) == "character"){
+    tau <- metaBMA::prior(family = tau, param = tau.par)
+  } else if (!class(tau) == "prior"){
+    stop("The argument 'tau' must be ",
+         "\n  (A) a character such as 'halfcauchy' specifying the family of prior distribution",
+         "\n  (B) a non-negative prior distribution specified via metaBMA::prior()")
+  }
+
```

Comment:
Can't tell from the test files and the codecov is not working for some reason. Are the following lines of code covered with tests?
https://github.com/IndrajeetPatil/ggstatsplot/blob/2a83c0f54e951b3613cefea06b08d1e554a7dad3/R/helpers_ggcoefstats.R#L851-L866